packages/views/file_upload.jade

```diff
@@ -0,0 +1,12 @@
+template(name="fileUpload")
+  .input-group
+    .input-group-btn
+      label.btn.btn-default.btn-file
+        span Choose file
+        i.fa.fa-folder-open
+        input.upload(name=name type="file")
+    .file-name-wrap
+      input.form-control.file-name(value="{{fileName}}" class="{{hasFile}}" readonly)
```

Comment:
Oh that's a good point too. But I actually meant that fileName contains the no file message sometimes instead of the file name. That could just be in the template and shown based on hasFile.